package.json

```diff
@@ -18,7 +18,9 @@
     "qbuild": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*' --ignore '@uma/common' --ignore '@uma/core'",
     "build": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '@uma/common' --ignore '@uma/core'",
     "clean": "yarn lerna run clean --stream",
-    "clean-packages": "yarn lerna clean --yes && rm -rf node_modules"
+    "clean-packages": "yarn lerna clean --yes && rm -rf node_modules",
+    "optimism-up": "./scripts/ovm/run-optimism.sh",
+    "optimism-down": "cd optimism/ops && docker-compose down -v"
```

Comment:
why do you have the `-v` in this command? does this not just print the version?

Reply:
```
    -v, --volumes           Remove named volumes declared in the `volumes`
                            section of the Compose file and anonymous volumes
                            attached to containers.
```

So sanity check to remove any volumes created, but i don't think there are any. I can remove to be clean